providers/openmetrics/client_interceptor.go

```diff
@@ -0,0 +1,36 @@
+package metrics
+
+import (
+	openmetrics "github.com/prometheus/client_golang/prometheus"
+	"google.golang.org/grpc"
+
+	"github.com/grpc-ecosystem/go-grpc-middleware/v2/interceptors"
+)
+
+var (
```

Comment:
Can we remove this? This is global we wanted to remove, no?

Reply:
Yeah, I didn't understand it before, removing it totally.